src/style.css

```diff
@@ -0,0 +1,117 @@
+
+@font-face {
+  font-family: "Avenir Book";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: "Book";
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-weight: normal;
+}
+
+/*styling simple html tags  */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir", Arial, sans-serif;
+  width: 100%;
+  font-weight: 300;
+  margin: 0 auto;
+  border: none;
+  padding: 0;
+}
+
+/* styles for container and wrappers*/
+.container {
+  width: 100%;
+  padding: 0 8px;
+  position: relative;
+}
+
+.wrapper {
+  margin-top: 20px;
+  position: relative;
+  background-color: white;
+  width: 100%;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+/*  styiling searchars */
+.searchbar_big {
+  background: #fff;
+  font-family: "Avenir Book", Arial, sans-serif;
+  min-width: 100%;
+  height: 70px;
+  font-size: 16px;
+  padding: 25px 0 23px 62px;
+  width: 100%;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  vertical-align: middle;
+  top: 20px;
+}
+
+.searchbar_small {
+  font-family: "Avenir Book", Arial, sans-serif;
+  background: #fff;
+  min-width: 100%;
+  height: 42px;
+  font-size: 14px;
+  line-height: 19px;
+  padding: 12px 0 12px 33px;
+  vertical-align: middle;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-weight: 300;
+  color: #3d4e61;
+}
```

Comment:
you repeat a lot of styles from the big class, go through [checkilst](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) and fix it